src/launchWizard/launchWizard.js

```diff
@@ -18,12 +18,62 @@
 // Retrieve vscode api - Doing this multiple times causes issues with the scripts
```

Comment:
Can you explain why there are both launchWizard.js and launchWizard.ts files? I thought vscode was typescript based, but I noticed some time ago that there is also a smaller portion of ".js". 

Any short explanation to motivate this would help me better understand the code. Up to you whether that is just a response to this comment, or becomes a comment in the code (somewhere, perhaps not here) or doc.

Reply:
So the reason there is both `launchWizard` .ts and .js is because the webview UI will directly communicate to JS scripts. However, we may need some data that only be gotten through the typescript, such as different npm packages. So the posting message stuff done in both JS and TS is how we communicate from the WebView JS to the TypeScript code.

Reply:
So to further explain for any custom webview or anything we make we need to make a JS and TS file. This is because JS is used for talking to the UI/DOM and then the TS is used for calculating, importing packages to run tasks, etc. We need it this way as it how the API is designed that only JS can communicate to the UI/DOM and thats basically all it can do. Then TS is the opposite, where it can do most things except access the UI directly. I hope this makes sense, let me know if doesn't I can try to explain it better.